specs/agents/tracing-transactions.md

```diff
@@ -4,4 +4,21 @@ Transactions are a special kind of span.
 They represent the entry into a service.
 They are sometimes also referred to as local roots or entry spans.
 
-Transactions are created either by the built-in auto-instrumentation or an agent or the [tracer API](tracing-api.md).
\ No newline at end of file
+Transactions are created either by the built-in auto-instrumentation or an agent or the [tracer API](tracing-api.md).
+
+#### Transaction outcome
+
+The `outcome` property denotes whether the transaction represents a success or a failure from the perspective of the entity that produced the event.
+The APM Server converts this to the [`event.outcome`](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-outcome.html) field.
+This property is optional to preserve backwards compatibility.
+If an agent doesn't report the `outcome` (or reports `null`), the APM Server sets the outcome to `"unknown"`.
```

Comment:
If its unknown then it won't be included in the error rate, right?

Reply:
yep

Reply:
I've clarified the definition of error rate: https://github.com/elastic/apm/pull/299/commits/362e2a1825788087d6128a271728d6c986b87662